components/tcf/secondLayer/src/settings.js

```diff
@@ -1,10 +1,23 @@
+export const ADEVINTA_COLLECTED_CONSENTS = {
```

Comment:
❗ 
remember to open a tech debt task for this, as it's not mentioned in __further considerations__